packages/ses/package.json

```
@@ -91,6 +100,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.29.1",
+    "hermes-engine-cli": "0.12.0",
```

Consider unpinning this?

```suggestion
    "hermes-engine-cli": "^0.12.0",
```

(Note: this will still require a bump for `0.13.0`, but will be satisfied by a hypothetical future `0.12.1`.)